tests/autoscaling/autoscaling_test.go

```diff
@@ -89,35 +88,20 @@ func (s *apiTestSuite) TestAPI(c *C) {
             "scale_in_interval_seconds":30
         }
     ],
-    "resource_expectations":[
-        {
-            "component":"tikv",
-            "cpu_expectation":4,
-            "count":8
-        },
-        {
-            "component":"tikv",
-            "cpu_expectation":8,
-            "count":2
-        },
-        {
-            "component":"tidb",
-            "cpu_expectation":2,
-            "count":2
-        }
-    ],
     "resources":[
         {
             "resource_type":"resource_a",
             "cpu":1,
             "memory":8,
-            "storage":1000
+            "storage":1000,
```

Comment:
The unit here is not the same as the definition

Reply:
It is only a test for testing the function of serialization.